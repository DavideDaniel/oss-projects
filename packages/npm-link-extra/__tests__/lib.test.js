const nleModule = require('../lib');

describe('npm-link-extra', () => {
  it('should match its snapshot', () => {
    expect(nleModule).toMatchSnapshot();
  });
});
