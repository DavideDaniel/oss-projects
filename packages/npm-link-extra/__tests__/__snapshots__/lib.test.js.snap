// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`npm-link-extra should match its snapshot 1`] = `
Object {
  "getDirectories": [Function],
  "getLinkedDeps": [Function],
  "getPackages": [Function],
  "getSharedDepDirs": [Function],
  "getSharedDeps": [Function],
  "getSharedLinked": [Function],
  "linkIfExists": [Function],
  "showLinkedDeps": [Function],
  "showSharedDeps": [Function],
  "unlinkIfLinked": [Function],
}
`;
